tests/pam/netconf.conf

```diff
@@ -0,0 +1,4 @@
+#%PAM-1.4
+auth required /home/jkt/work/prog/_build/czechlight-clang14-asan-ubsan-ly2/libnetconf2/tests/pam_netconf.so
+account required /home/jkt/work/prog/_build/czechlight-clang14-asan-ubsan-ly2/libnetconf2/tests/pam_netconf.so
+password required /home/jkt/work/prog/_build/czechlight-clang14-asan-ubsan-ly2/libnetconf2/tests/pam_netconf.so
```

Comment:
remove the file from the commit

Reply:
Oops, I was too lazy with the `git add`. That's the danger of generating files in the src dir :).